lib/gettext-setup/version.rb

```diff
@@ -0,0 +1,3 @@
+module GettextSetup
+  VERSION = '0.24'
```

Comment:
You were too quick, I just submitted another pull request which should justify what I am trying to achieve here, see puppetlabs/puppet#5822.